airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java

```diff
@@ -195,6 +237,19 @@ public void setDestinationInstancewideOauthParams(final SetInstancewideDestinati
     configRepository.writeDestinationOAuthParam(param);
   }
 
+  private JsonNode getOAuthInputConfigurationForConsent(final ConnectorSpecification spec,
+                                                    final JsonNode hydratedSourceConnectionConfiguration,
+                                                    final JsonNode destinationDefinitionIdRequestBody) {
```

Comment:
nit: rename this last parameter to oAuthInputConfiguration, since this current parameter name is not really quite right

Reply:
done